### ISSUE TITLE
[V6][Bug] theme-tabler menu spacing is wrong

### ISSUE THREAD

**Issue by fronbow** [Laravel-Backpack/CRUD#5117]
# Bug report

### What I did

Upgraded/Updated to v6

### What I expected to happen

My side menu to just work as it did previously

### What happened

The spacing between menu items is bigger

### What I've already tried to fix it

Had a look at the monster demo for v6 and the sub-menu items aren't semantic - they're divs rather than list items, but they line up and look perfect

### Backpack, Laravel, PHP, DB version

When I run ```php artisan backpack:version``` the output is:
### PHP VERSION:
PHP 8.1.12-1ubuntu4 (cli) (built: Feb 22 2023 19:48:21) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.12, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.12-1ubuntu4, Copyright (c), by Zend Technologies
    with Xdebug v3.2.0, Copyright (c) 2002-2022, by Derick Rethans

### LARAVEL VERSION:
10.13.5.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 0.16.0
backpack/crud: v6.x-dev
backpack/generators: v6.x-dev
backpack/permissionmanager: v6.x-dev
backpack/pro: v6.x-dev
backpack/theme-tabler: dev-main

Cheers

**Comment by tabacitu** [comment]
Hey @fronbow ,

Thanks a lot for submitting this. I agree - there were inconsistencies between themes - some had menu items with `divs`, some had `li`s, not to mention the classes were all different.

At first we thought we can keep one HTML structure, then for each theme create CSS for that structure so it looks good. But that fell apart after a few themes. It worked _most of the time_, but... it was shitty to do, and we could easily see the scenario where a theme would just require a totally different HTML structure to achieve what they want.

So we've created a new way to define menu items & dropdown items in Backpack v6 - #5135 - and you can see their docs here - https://backpackforlaravel.com/docs/6.x/base-components#menu-item-1

Basically now the developer is supposed to use these x-components. Then each theme turns that component into the HTML structure the theme wants. That way, it's the theme's responsibility, not the developer's.

Thanks again for taking the time to submit these reports. Even though we haven't answered right away, we did take them int consideration, talked about them and found solution for every one of them 😀 I'm going through them now to answer each one. Please keep submitting if you find stuff.

Cheers!
